src/models/Request.model.ts

```
@@ -0,0 +1,44 @@
+        },
+        status: {
+            type: String,
+            enum: ['pending', 'approved', 'rejected'],
+            default: 'pending'
+        }
+    },
+    {
+        timestamps: true,
+        toJSON: {
+            transform: (_, ret) => {
+                delete ret.__v;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)